plugins/modules/ini_file.py

```diff
@@ -116,6 +116,13 @@
       - Allow option without value and without '=' symbol.
     type: bool
     default: false
+  modify_inactive_option:
+    description:
+      - Do not replace a commented line that matches the given option. This is useful when you want to keep example
+       key:value pairs for documentation purposes.
+    type: bool
+    default: true
+    version_added: 7.6.0
```

Comment:
The next release with new features will be 8.0.0:
```suggestion
    version_added: 8.0.0
```